tests/integration_tests/functional/test_drive_vhost_user.py

```diff
@@ -8,16 +8,14 @@
 
 from framework import utils
 
-VHOST_USER_SOCKET = "/vub.socket"
 
-
-def spawn_vhost_user_backend(vm, host_mem_path):
-    """Spawn vhost-user-block backend."""
+def spawn_vhost_user_backend(vm, host_mem_path, socket_path):
```

Comment:
could be good to explain in the commit message why we do this.

Reply:
added explanation